cli/parser/options/options.go

```diff
@@ -279,7 +274,10 @@ type Option interface {
 	SetValue(string)
 	GetDefinition() *Definition
 	SetDefinition(*Definition)
-	UseShortName(bool)
+	UseName()
+	UseShortName()
+	UsesName() bool
```

Comment:
nit: imo `UseName` / `UsesName` would be clearer as `UseFullName` / `UsesFullName`.

Reply:
The problem I was having with that is that both of those also sound like they could apply to `OldName`. I'll switch it to use `Standard` to match the flag forms.

Reply:
I started doing this, and as I was cascading the change from `name` -> `standardName` into the option definition, it felt really wrong. In the bazel flags proto it's called the `name` and we've used that everywhere else. The idea is that (for now, at least) the three options are "Name", "ShortName", and "OldName", which all correspond directly to fields in the bazel flags proto.